proto/lavanet/lava/pairing/relay.proto

```diff
@@ -85,12 +85,12 @@ message RelayRequest {
 
 message RelayReply {
     bytes data = 1;
-    bytes sig = 2; // sign the data hash+query hash+nonce
```

Comment:
Is the comment true though? From `relay_exchange.go`:
```
func (re RelayExchange) DataToSign() []byte {
	re.Reply.Sig = nil
	var metadataBytes []byte
	for _, metadata := range re.Reply.GetMetadata() {
		data, err := metadata.Marshal()
		if err != nil {
			utils.LavaFormatError("metadata can't be marshaled to bytes", err)
		}
		metadataBytes = append(metadataBytes, data...)
	}
	// we remove the salt from the signature because it can be different
	re.Request.RelayData.Salt = nil
	msgParts := [][]byte{
		re.Reply.GetData(),
		[]byte(re.Request.RelayData.String()),
		metadataBytes,
	}

	return sigs.Join(msgParts)
}
```

It looks like we sign on the Reply Data, the Request's Relay Data and the Metadata. Right?